lms/models/registration.py

```diff
@@ -0,0 +1,15 @@
+import sqlalchemy as sa
+
+from lms.db import BASE
+
+
+class Registration(BASE):
```

Comment:
This identifies a "client_id". All deployment_id link the the same client_id share the same auth credentials.